### ISSUE TITLE
Write tests for loading rooms into game_state structs

### ISSUE THREAD

**Issue by mewthew** [uchicago-cs/chiventure#166]
After we finish design and implementation for loading rooms in #140, we need to ensure that our code correctly translates libobj room structs into game_state structs. We will write unit tests that cover the `add_rooms_to_game` and `add_connections_to_rooms` functions to make sure rooms are successfully created and added to a given game pointer.

**Comment by mewthew** [comment]
We have finished the implementation for some basic testing of loading libobj structs into game-state structs, which can be found [here](https://github.com/uchicago-cs/chiventure/blob/wdl/load_rooms/src/wdl/tests/test_rooms.c). In summary, our tests assert whether `add_rooms_to_game` and `add_connections_to_rooms` succeeded, and we also used `strncmp` to check whether certain fields (e.g. short_desc) were correctly implemented in the game-state struct through `find_room` and `get_sdesc` game-state functions.

NOTE: our functions aren't currently testable due to game-state not having their latest code pushed into master. Hence, we will make sure that our implementation is valid next sprint and upgrade our tests such that they are more automated and test more attribute fields within the game-state structs.
